js/app.js

```diff
@@ -1984,7 +1984,7 @@ var _game_loop = function() {
 					heat_add += tile.heat;
 					tile.setTicks(tile.ticks - 1);
 
-					if ( tile.reflectors.size ) {
+					if ( tile.reflectors.length ) {
```

Comment:
size was from the Set object, must have replaced it at some point and missed it here